kubernetes/README.md

```diff
@@ -1,6 +1,6 @@
 # Deploy 1Password SCIM Bridge on Kubernetes
 
-*Learn how to deploy 1Password SCIM Bridge on a Kubernetes cluster.*
+_Learn how to deploy 1Password SCIM Bridge on a Kubernetes cluster._
```

Comment:
These autoformatting changes are actually changing bolded statements to italicized, which we don't want. Can we change back to using `*` ?

Reply:
Comparing with our [main branch](https://github.com/1Password/scim-examples/tree/master/kubernetes) the text initially represented with *xxx* as italics are now adapted as  _xxx_ by editor. Do we want to make it bold there?